packaging/ondemand.spec

```diff
@@ -322,7 +322,8 @@ touch %{_localstatedir}/www/ood/apps/sys/activejobs/tmp/restart.txt
 touch %{_localstatedir}/www/ood/apps/sys/myjobs/tmp/restart.txt
 
 # Rebuild Apache config and restart Apache httpd if config changed
-if /opt/ood/ood-portal-generator/sbin/update_ood_portal --rpm ; then
+/opt/ood/ood-portal-generator/sbin/update_ood_portal --rpm --detailed-exitcodes
+if [[ $? -eq 2 ]] ; then
```

Comment:
Make that `-eq 3` and update `ood-portal-generator` to clarify Exit with 3:

https://github.com/OSC/ondemand/blob/a81709151fab023bbed8ec058d0c0edd9626ead5/ood-portal-generator/sbin/update_ood_portal#L22

Reply:
Ya I fixed that and spec logic.